tools/test-expect-version/test-expect-version.c

```diff
@@ -44,11 +44,14 @@
 #define MSGLEN      (4 + 4 + 8)
 
 #ifndef UART_DEV
-#define UART_DEV "/dev/ttyS0"
+#define UART_DEV "/dev/ttyACM1"
```

Comment:
This breaks the jenkins tests, please revert to "/dev/ttyS0"

Reply:
reverted